akvo/rsr/spa/webpack.config.js

```diff
@@ -26,7 +27,7 @@ const config = () => {
     output: {
       filename: 'bundle.js',
       path: resolve(__dirname, 'dist'),
-      publicPath: '',
+      publicPath: '/my-rsr/',

```

Comment:
Why have you changed this?

Reply:
Because otherwise the links don't reach the webpack server. Our webpack servers + nginx configuration requires us to have URLs prefixed with /my-rsr/ or /project-directory/ to hit the webpack server.